src/test/java/io/jenkins/plugins/testing/DotCoverStepExecutionTest.java

```diff
@@ -71,7 +71,7 @@ public void testInvalidTestDllFailsBuild() throws Exception // TODO can i get th
         CpsFlowDefinition pipelineDefinition = new CpsFlowDefinition(pipeline, true);
         project.setDefinition(pipelineDefinition);
         WorkflowRun build = project.scheduleBuild2(0).get(); // schedule right now and wait to complete.
-        master.assertBuildStatus(Result.FAILURE, build);
+        master.assertBuildStatus(Result.SUCCESS, build);
```

Comment:
This test is meant to fail the build.
So, something changed and needs updating to make sure our assumptions are correct.